src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -3162,11 +3154,11 @@
                     <ControlTemplate.Triggers>
                         <Trigger Property="IsMouseOver" Value="True">
                             <Setter TargetName="border" Property="Opacity" Value="0.9" />
-                            <Setter TargetName="mouseOverBorder" Property="Visibility " Value="Visible" />
```

Comment:
I believe these are just typos. I was getting warnings in VS because of the extra empty space.